Razorpay.nuspec

```diff
@@ -14,11 +14,12 @@
     <copyright>Copyright 2015</copyright>
     <tags>Razorpay</tags>
     <dependencies>
-      <dependency id="Newtonsoft.Json" version="11.0.2" />
+      <dependency id="Newtonsoft.Json" version="[11.0.2,13.0.3)" />
```

Comment:
closing Brackets are not same. Please test

Reply:
using single version instead of multiple , also checked the `v13.0.3` is compatible for old .net version as well.

![Screenshot 2023-09-13 114701](https://github.com/razorpay/razorpay-dot-net/assets/89454448/7cf69b09-31f8-4ff1-bde2-2a7e49cd3ab4)